src/api.js

```diff
@@ -1,15 +1,25 @@
-const API_URL = 'https://bootcamp-api.codeit.kr/api/sample';
+const API_URL = 'https://bootcamp-api.codeit.kr/api';
 
-export async function getProfile() {
-  const response = await fetch(`${API_URL}/user`);
-  const body = await response.json();
+export function getSampleUser() {
+  return fetch(`${API_URL}/sample/user`).then(res => res.json());
+}
+
+export function getSampleFolder() {
+  return fetch(`${API_URL}/sample/folder`).then(res => res.json());
+}
 
-  return body;
+export function getUsers() {
+  return fetch(`${API_URL}/users/1`).then(res => res.json());
 }
 
-export async function getFolder() {
-  const response = await fetch(`${API_URL}/folder`);
-  const body = await response.json();
+export function getSortingTab() {
+  return fetch(`${API_URL}/users/1/folders`).then(res => res.json());
+}
+
+export function getFolders() {
+  return fetch(`${API_URL}/users/1/links`).then(res => res.json());
+}

```

Comment:
제안드립니다
- 함수 이름은 엔드포인트와 맞춰주시면, 오해없이 편하게 쓸 수 있을거같아요
- userId가 1로 고정되는건 - 미션때문이지만-  아주 강한 가정이니까 api자체 보다는 사용하는 쪽에 드러날 수 있게, 외부에서 인자로 받으면 좋겠습니다

```suggestion
export function getUserLinks(userId) {
  return fetch(`${API_URL}/users/${userId}/links`).then(res => res.json());
}

```